### DISCUSSION TITLE
why use argon2 cryptographic hash algorithm, not bcrypt?

### DISCUSSION THREAD

**Discussion by anlexN** [directus/directus#15583] [Archive]
please give me a good explanation.

**Comment by rijkvanzanten** [comment]
Cause it's flexible and generally speaking harder to brute force if you have access to the hashes in a breach. That being said, I'd invite you to do your own research on the topic, and let me know what you find 👍🏻

**Reply by anlexN** [reply]
your say brute force is `Timing Attacks`, right?
